### PR TITLE
Add power poles gracethd France merge analyzers, starting on Dordogne opendata

### PR DESCRIPTION
**Pull request by flacombe**
Earlier this year, some public authorities started publishing opendata about power poles used to support fibre optic roll out.

I'm proposing here a merge analysis based upon graceTHD standard, implemented on Dordogne department.  
Data quality is really good and this area is proposed with 5m as conflation distance.

It is supposed to raise 11 697 warnings and currently conflates with 16 osm existing nodes.
For instance https://osm.org/node/11285907938 accurately matches with `POT-24253-BT-07047` opendata record.

@frodrigo we should adjust classes and item numbers please.

### CODE REVIEW

File: analysers/analyser_merge_power_pole_FR_gracethd_dordogne.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        Analyser_Merge_power_pole_FR_gracethd.__init__(self, config,
+            source_url='https://www.data.gouv.fr/fr/datasets/appuis-aeriens-enedis-utilises-dans-le-cadre-du-deploiement-de-la-fibre-sur-le-rip-de-la-dordogne/',
+            dataset_name='Appuis aériens ENEDIS utilisés dans le cadre du déploiement de la Fibre sur le RIP de la Dordogne',
+            source=SourceDataGouv(
+                attribution="Syndicat Mixte Périgord Numérique",
+                dataset="659d72fb641c7c0d6fe6cc59",
+                resource="82e49c1f-976f-4be7-ab20-0a58e9badb56"),
+            srid = 2154,
+            conflationDistance=5,
+            classs=1000,
```
Comment (frodrigo) [inline analysers/analyser_merge_power_pole_FR_gracethd_dordogne.py:37]: Class 1000 is already in use for street light.
https://osmose.openstreetmap.fr/fr/issues/open?item=8490

Comment (flacombe) [reply analysers/analyser_merge_power_pole_FR_gracethd_dordogne.py:37]: Yes I know.
Which one I should use?

**Comment by frodrigo** [comment]
2000

**Comment by flacombe** [comment]
Switched to 8290/8291 for power supports

**Comment by frodrigo** [comment]
Thank you. Merged, but not deployed, yet.

**Comment by frodrigo** [comment]
Results
https://osmose.openstreetmap.fr/fr/issues/open?source=448210&item=xxxx

**Pull request merged**